expected/13.12/extra/aggregates.out

```diff
@@ -994,13 +1173,13 @@ CREATE FOREIGN TABLE bitwise_test(
 --Testcase 43:
 SELECT
   BIT_AND(i2) AS "?",
-  BIT_OR(i4)  AS "?"
+  BIT_OR(i4)  AS "?",
+  BIT_XOR(i8) AS "?"
 FROM bitwise_test;
- ? | ? 
----+---
-   |  
-(1 row)
-
+ERROR:  function bit_xor(bigint) does not exist
+LINE 4:   BIT_XOR(i8) AS "?"
```

Comment:
is it bit_xor(bigint), bit_xor(smallint) only support postgres version > 13.12?

Reply:
No, I have read this documentation:
- https://www.postgresql.org/docs/13/functions-aggregate.html
- https://www.postgresql.org/docs/14/functions-aggregate.html

The difference is between PostgreSQL 13 and 14

Reply:
Thank you for supporting. It's OK